### PR TITLE
Revision of the Spanish translation of GTFS-realtime docs

### LINKED ISSUE #54: Revision of the current Spanish translation of GTFS-realtime docs
Hi everyone,

I've been working with GTFS recently and I've realized that the Spanish translation of the GTFS-realtime spec might need a revision.
If nobody else is currently working on it, as a native Spanish speaker **I wouldn't mind doing it myself** in the following days/weeks; but I've been advised to ask here first just in case somebody's already on it.
I'm looking forward to reading your opinions,

Eduardo

### PR DESCRIPTION
**Pull request by eduherminio**
Fix a number of translation inaccuracies.
Add Occupancy status (missing in the Spanish version of vehicle-positions.md).
Add a missing paragraph to the English version of reference.md.

Resolves #54.

### CODE REVIEW

**Comment by eduherminio** [comment]
Are there any plans to merge this, @RachM, or may I just close it?

**Comment by RachM** [comment]
Thanks for taking the time to translate - I've reviewed and will merge.

File: gtfs-realtime/spec/en/reference.md [hunk 1]
```diff
@@ -73,6 +73,13 @@ Fields labeled as **experimental** are subject to change and not yet formally ad
 
 The contents of a feed message. Each message in the stream is obtained as a response to an appropriate HTTP GET request. A realtime feed is always defined with relation to an existing GTFS feed. All the entity ids are resolved with respect to the GTFS feed.
 
+A feed depends on these external configurations:
```
Comment (RachM) [inline gtfs-realtime/spec/en/reference.md]: I don't think we need this paragraph - it doesn't add much value.

Comment (eduherminio) [reply gtfs-realtime/spec/en/reference.md]: I'm removing it this evening :)

**Pull request merged**